### ISSUE TITLE
Using just the ID it does not work

### ISSUE THREAD

**Issue by dariocavada** [delay/getx_google_sheets_translation_code_generator#2]
Test your code and with the id provided in the `main.dart` it works,
copying the file and using the other generated id it does not work...

I find a way to solve this.
If you publish the file to the web it is possible to select the output format and the sheet, it generates then the correct url to put in the `main.dart` file.

Esample:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/4304176/174007594-5ff822e6-ac64-4e96-bb24-367964ae20c8.png">

**Comment by delay** [comment]
Yes you would need to make the url to your translation viewable in read format so the app can see it when called from your app.  I need to update the instructions and add this.  

Thanks for reporting this trouble!
